src/client/rest/RESTMethods.js

```diff
@@ -17,11 +18,13 @@ class RESTMethods {
     this.rest = restManager;
   }
 
-  login(token = this.rest.client.token) {
+  get client() { return this.rest.client; }
```

Comment:
Don't do this purely for laziness. Getters will have a slight performance overhead vs. accessing an actual property.